bib/internal/buildconfig/config.go

```diff
@@ -1,26 +1,45 @@
 package buildconfig
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 
 	"github.com/pelletier/go-toml"
+	"github.com/sirupsen/logrus"
 
 	"github.com/osbuild/images/pkg/blueprint"
 )
 
-type BuildConfig struct {
-	Blueprint *blueprint.Blueprint `json:"blueprint,omitempty" toml:"blueprint"`
+type legacyBuildConfig struct {
+	Blueprint *json.RawMessage `json:"blueprint"`
 }
 
+type BuildConfig blueprint.Blueprint
```

Comment:
We could just use blueprint.Blueprint everywhere instead of build config, it will make the diff a bit bigger this is why I went with the type alias but I have no super strong opinions here, either way is fine for me.